src/CMakeLists.txt

```diff
@@ -57,11 +57,13 @@ include_directories(
 if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
   set(CMAKE_C_FLAGS
     "${CMAKE_C_FLAGS} -Wall -Wstrict-prototypes -Wpointer-arith")
+  set(CMAKE_C_FLAGS_DEBUG "-O0")
   set(CMAKE_C_FLAGS_RELEASE "-O3")
   set(CMAKE_C_FLAGS_RELWITHDEBINFO "-O2 -g")
 
   set(CMAKE_CXX_FLAGS
     "${CMAKE_CXX_FLAGS} -Wall -Wpointer-arith")
+  set(CMAKE_CXX_FLAGS_RELEASE "-O0")
```

Comment:
I guess this should be CMAKE_CXX_FLAGS_DEBUG but is it really necessary?  Isn't it the default?